### DISCUSSION TITLE
Running with nginx on same machine

### DISCUSSION THREAD

**Discussion by NoodlesDev987** [NginxProxyManager/nginx-proxy-manager#1345] [Q&A]
Hello, is it possible to run this when you have nginx already on the server and you don't have money to het another server for npm.

I have tried (with ports 8080, 8181, 4443), installed it, the gui works. I have created proxy host, but I can't create SSL cert (Internal error). When I go to proxy host domain, I get default nginx 404 error.

How to fix?

**Comment by jc21** [comment]
If you're not exposing port 80, then letsencrypt cannot use http (default) validation when requesting a new certificate.

You will have to use a dns method when requesting a certificate to get around this.

Alternatively, you can give your server 2 or more IP addresses and have your other server listen/bind to one of them and have npm bind to the other one. Then you can use port 80 on both but this is a very advanced idea and support for that is out of scope for this project.
